### PR TITLE
Update toast.is-a.dev for GitHub Pages domain verifications

### PR DESCRIPTION
**Pull request by lightly-toasted**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://toast.is-a.dev/

### CODE REVIEW

**Comment by lightly-toasted** [comment]
Why did it fail?

File: domains/_github-pages-challenge-lightly-toasted.toast.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "github-username",
```
Comment (Stef-00012) [inline domains/_github-pages-challenge-lightly-toasted.toast.json:3]: You should put your github username

File: domains/_github-pages-challenge-lightly-toasted.toast.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        "email": "email@address"
```
Comment (Stef-00012) [inline domains/_github-pages-challenge-lightly-toasted.toast.json:4]: You should put your email address

File: domains/toast.json [hunk 3]
```diff
@@ -6,6 +6,11 @@
         "email": "tooast@duck.com"
     },
     "record": {
-        "CNAME": "lightly-toasted.github.io"
+        "CNAME": "lightly-toasted.github.io",
```
Comment (Stef-00012) [inline domains/toast.json:9]: You can't use other records with CNAME

**Review by Stef-DP — DISMISSED** [review]

**Comment by lightly-toasted** [comment]
Thanks.

**Review by wdhdev — APPROVED** [review]

**Pull request merged**